omero/sysadmins/unix/install-web/install-wsgi.txt

```diff
@@ -89,6 +89,17 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+Additional settings can be configured from command line arguments:
```

Comment:
@hflynn this is an output of web start -h if we have any better way to format it go for it.